ReticentJohn/Amaroq
# Feature: Adds sensitivity toggle option

# Summary

Someone was asking for a global flag to toggle the visibility of sensitive contents on here (https://github.com/ReticentJohn/Amaroq/issues/100). So this PR does exactly that.

### Changes
- Adds new image asset. (an eye icon)
- New flag for toggling visibility for sensitive contents on the DWSettingStore.

### Caveat
- I took the liberty to run a update on the app's pods. So most of the changes come from those 3rd party frameworks. The updates shouldn't affect anything, it's just nice to keep things updated.

### How to test the new changes
- Post some text and a image and then make sure to mark them as sensitive.
- Go to Menu (that last tab on the lower right) -> App Settings -> try toggle "Enable NSFW mode globally"
- Check out that new toot you just posted.
